ledger-service/http-json-perf/BUILD.bazel

```diff
@@ -124,24 +100,7 @@ daml_compile(
             "@maven//:io_gatling_highcharts_gatling_charts_highcharts",
         ] + perf_runtime_deps.get(edition),
         deps = [
-            "@maven//:io_gatling_highcharts_gatling_charts_highcharts",
-            "@maven//:org_scalatest_scalatest_compatible",
             ":http-json-perf-{}".format(edition),
-            "//language-support/scala/bindings-akka",
-            "//ledger-api/rs-grpc-bridge",
-            "//ledger-service/http-json:http-json-{}".format(edition),
-            "//ledger-service/http-json-testing:{}".format(edition),
-            "//libs-scala/gatling-utils",
-            "//libs-scala/jwt",
-            "//libs-scala/scala-utils",
-            "@maven//:com_fasterxml_jackson_core_jackson_core",
-            "@maven//:com_fasterxml_jackson_core_jackson_databind",
-            "@maven//:io_gatling_gatling_app",
-            "@maven//:io_gatling_gatling_commons",
-            "@maven//:io_gatling_gatling_core",
-            "@maven//:io_gatling_gatling_http",
-            "@maven//:io_gatling_gatling_http_client",
-            "@maven//:org_slf4j_slf4j_api",
```

Comment:
We only need `http-json-perf-{}` here, the rest are just copies from that library.